src/passes/Print.cpp

```diff
@@ -1440,7 +1440,10 @@ struct PrintExpressionContents
         break;
     }
   }
-  void visitRefNull(RefNull* curr) { printMedium(o, "ref.null"); }
+  void visitRefNull(RefNull* curr) {
+    printMedium(o, "ref.null ");
+    o << curr->type.getHeapType();
```

Comment:
Here is where `(ref.null extern)` is printed. Was `(ref.null)`, and I'm not sure whether the spec wants `(ref.null externref)` instead?

Reply:
You have it right as `(ref.null extern)`